main/reference/zoe-api/mutable-quote.md

```diff
@@ -0,0 +1,28 @@
+# NOT CURRENTLY INCLUDED IN THE PUBLISHED DOCS; PLEASE IGNORE
```

Comment:
So we're supposed to store such temporarily files locally, rather than in the cloud-based git repo? Well, OK, but that seems to defeat a major purpose of git repos.